Framework/include/QualityControl/ProvenanceToQCRunType.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   ProvenanceToQCRunType.h
+/// \author Michal Tichak
+///
+
+#include <stdexcept>
+#include <string>
+namespace o2::quality_control::core
+{
+
+enum class QCRunType {
+  Data,
+  Simulation
+};
+
+inline QCRunType toRunType(const std::string& provenance)
```

Comment:
We call run type those: https://github.com/AliceO2Group/Control/blob/f894c3c8b1b7597fed5bc6281d0b195bc7e088cd/common/runtype/runtype.go#L37

Perhaps we could just have a converter from string to enumerated Provenance? Then you could handle `Provenance::SyncQC` and `Provenance::AsyncQC` the same way when sending flags to BKP.